crates/npmrc/src/lib.rs

```diff
@@ -229,8 +229,9 @@ mod tests {
     #[test]
     pub fn should_use_pacquet_home_env_var() {
         env::set_var("PACQUET_HOME", "/hello");
+        let home_dir = dirs::home_dir().unwrap();
         let value: Npmrc = serde_ini::from_str("").unwrap();
-        assert_eq!(value.store_dir, PathBuf::from_str("/hello/store").unwrap());
+        assert_eq!(value.store_dir, home_dir.join("/hello/store"));
```

Comment:
`home_dir.join("/hello/store")` would just be `/hello/store`. There is something wrong with either this test or the implementation.

Reply:
Actually, just do this:

```suggestion
        assert_eq!(value.store_dir, Path::new("/hello/store"));
```